high-output-management/chapters1-2.tex

```diff
@@ -0,0 +1,133 @@
+
+\section{Indicators}
+
+Indicators enable appropriate and accurate forecasting for decision making, such as:
+
+\begin{enumerate}
+\item Sales forecasts
+\item Manpower quantity and quality
+\item Equipment status and output rate
+\item Raw materials
+\end{enumerate}
+
+Taken on their own, indicators can result in a negative feedback loop. Paired indicators resolve Goodhart’s law-like bias from metrics. 
+
+\begin{itemize}
+\item For instance, suppose you’re measuring custodial staff by square feet cleaned alone. This may cause rooms to be poorly cleaned. By adding a complementary indicator, like the number of reports of unclean rooms, you can properly account for quality.
+\item Analogously, if you only record the number of shortages, then you may over-supply your inventory to avoid shortages. But if you also record end-of-day remaining inventory then you know not to over-invest.
+\item In software, consider using both the number of PRs (``diffs'' or ``branches'') merged as well as per-PR SLOC as a measure of developer productivity (neither is perfectly accurate, but both should be considered).
+\end{itemize}
+
+Provide variance estimates on measurements. Not every number should be taken under equal consideration. With variance estimates, you can add slack to scale with desired confidence in allocations. Critical processes may need more slack.
+
+In trend prediction,\footnote{This section is pretty weird, kind of like Andy just teaching you forecasting.} make sure that you use:
+\begin{itemize}
+\item Historical data
+\item Capture seasonality
+\item Autoregressive trends
+\item Nonlinearity (make sure you know what kind you need to have in your models)
+\end{itemize}
+
+Properly taking advantage of trends can lead to a lot of value creation (see the \nurl{https://www.cnbc.com/2018/05/03/how-ray-dalio-helped-launch-mcdonalds-chicken-mcnugget.html}{Bridgewater example}).
+
+\section{De-risking Parallel Flows}
+
+If you have two dependent flows that are both deep and need to converge to create value, then you're undertaking a lot of risk. For instance, consider some manufacturing process that transforms raw material into goods and an analogous sales process that are both required to start early (for long sales cycles) that result into a successful deal (see p.26 of the book for a good picture).
+
+Fail early. Capture failures at the lowest value stage (earlier) that is possible to detect them (needs to be traded off with detection costs). For instance, in candidate recruitment: assess candidate interest early in the pipeline, and if they’re just shopping around then they’re not worth the interview time investment.
+
+Introduce synchronization points as much as possible between parallel flows. Two parallel but dependent features that two teams are working should be incrementalized and interacting at as early stages as possible (rather than build two things separately then connect). An example in code development would be to rebase often.
```

Comment:
```suggestion
Introduce synchronization points as much as possible between parallel flows. Two parallel but dependent features that two teams are working should be incrementalized and should be interacting at as early stages as possible (rather than build two things separately then connect). An example in code development would be to rebase often.
```